keras-dreambooth-sprint/README.md

```diff
@@ -1,95 +1,135 @@
 
-[Link to notebook](https://colab.research.google.com/drive/1Bh49YBYgaNc3X5aRzaMZNska4ErvU3lJ?usp=sharing) 
+Add the category with their IDs to the model cards for submission and add `keras-dreambooth` to model card metadata in tags section. Here's an example [model card](https://huggingface.co/spaces/keras-dreambooth/example-submission/blob/main/README.md). All the submissions will be populated [in this leaderboard](https://huggingface.co/spaces/keras-dreambooth/leaderboard) and ranked according to likes on a given Space to determine the winners.
 
-You can fine-tune on any concept that you want. Couple of inspirations for you: 
+Couple of inspirations for you: 
 
+TODO: it might be confusing to show examples not using keras. maybe worth for us to build some of the examples and share them
 1. Lowpoly World: This [model](https://huggingface.co/MirageML/lowpoly-world) generates lowpoly worlds 🤯🌍
 2. Future Diffusion: This [model](https://huggingface.co/nitrosocke/Future-Diffusion) generates images in futuristic sci-fi concepts 🤖
 3. Fantasy sword: This [model](https://huggingface.co/MirageML/fantasy-sword) generates swords for fantasy themed games 🧙‍♂️
 
-Are you looking to stand out from the crowd? If you partner up with an artist to fine-tune on a **********consentful********** concept, you’ll be eligible to submit your model under the `consentful` theme!
-
-If you need more pointers on Dreambooth implementation with Keras, you can check out [this repository](https://github.com/sayakpaul/dreambooth-keras). 
+## Compute
 
 [ADD DETAILS ABOUT COMPUTE SPONSORSHIP]
 
-### Dreambooth Diffusers Integration with KerasCV
+## Model Training
 
-As of now, inference and deployment options of `KerasCV` is limited, which `diffusers` library could come for a rescue. With only few lines of code, we can convert a `KerasCV` model into a `diffusers` one and use `diffusers`’ pipelines to perform inference. You can get more information [here](https://huggingface.co/docs/diffusers/main/en/using-diffusers/kerascv). Also check out [this Space](https://huggingface.co/spaces/sayakpaul/convert-kerascv-sd-diffusers) for converting your `KerasCV` model to `diffusers`one.
+You can find the notebook here and adjust it according to your own dataset 👇  
 
-```
-from diffusers import StableDiffusionPipeline
+TODO: The notebook below is a Google Drive. We should add it to the GitHub repo
 
-# checkpoint of the converted Stable Diffusion from KerasCV
-model_ckpt = "sayakpaul/text-unet-dogs-kerascv_sd_diffusers_pipeline"
-pipeline = StableDiffusionPipeline.from_pretrained(model_ckpt)
-pipeline.to("cuda")
+[Link to notebook](https://colab.research.google.com/drive/1Bh49YBYgaNc3X5aRzaMZNska4ErvU3lJ?usp=sharing). If you need more pointers on Dreambooth implementation with Keras, you can check out [this repository](https://github.com/sayakpaul/dreambooth-keras).
```

Comment:
I'd love it if you could discard this change and keep the guide informative.

I don't want people to jump to other places.

Reply:
This code is still in line 109 in the "Dreambooth Diffusers Integration with KerasCV" section